providers/manage.rb

```diff
@@ -111,10 +111,12 @@ def chef_solo_search_installed?
       if manage_home_files?(home_dir, u['username'])
         Chef::Log.debug("Managing home files for #{u['username']}")
 
-        directory "#{home_dir}/.ssh" do
-          owner u['username']
-          group u['gid'] || u['username']
-          mode "0700"
+        if u['ssh_keys'] || u['ssh_private_key'] || ['ssh_public_key']
```

Comment:
`['ssh_public_key']`will always be true. I think you meant `u['ssh_public_key']` here?

We don't have any tests around this provider other than attempting to use it in the test recipe, but tests would be great :smile_cat: